I/IVerilog/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "IVerilog"
+version = v"11.0.0"
+
+# Collection of sources required to complete build
+sources = [
+   GitSource("https://github.com/steveicarus/iverilog.git", "84b4ebee0cfcda28a242d89a07020cd70b1d3e7f")
+]
+
+dependencies = [
+    Dependency("Bison_jll"; compat="~3.5.2"),
```

Comment:
Probably also bison is a host tool?
```suggestion
    HostBuildDependency("Bison_jll"),
```
?

Reply:
Somewhat certain. I didn't understand the `HostBuildDependency` until now.